lib/Simulation/Simulator.py

```diff
@@ -236,41 +297,35 @@ def summarize(self):
 
             
     def extract(self):
-        current_time = datetime.datetime.now()
-        path = current_time.strftime("%Y%m%d-%H%M")
-        os.mkdir(path)
-        
+        # This function always open and close the files 
+        # to guarantee that they are being rewriten from scratch
 
-        with open(join(path,'infection_summary.csv'), 'w', newline='') as csvfile:
-            fieldnames = ['Day','Hour','Minutes','CurrentStep','Susceptible','Exposed','Infectious','Recovered']
-            writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
+        summaryFilePath = join(self.reportPath,'infection_summary.csv')
+        with open(summaryFilePath, 'w', newline='') as summaryFile:
+            writer = csv.DictWriter(summaryFile, fieldnames=summaryFieldnames)
             writer.writeheader()
             for x in self.infectionHistory:
                 writer.writerow(x)
```

Comment:
TODO: Remember where in the "infection history" we stopped saving the log, and continue later.

Possible way to do this even better:
- every time we add a line to "infection history", we copy the dict to a variable named "logbuffer". 
- At "extract", we write "logbuffer" to the log, and clear the log